### PR TITLE
We are now escaping many template fields

### PR DESCRIPTION
**Pull request by annando**
Much work is still needed to really escape all fields.

### CODE REVIEW

**Comment by MrPetovan** [comment]
We need a tracking document for this, I guess I'm creating a shared spreadsheet again.

**Comment by annando** [comment]
How do you want to track it?

**Comment by annando** [comment]
I guess this PR here covers many places now. The next I would like to do on upcoming PRs.

File: mod/acl.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ function acl_content(App $a)
 			$groups[] = [
 				'type'  => 'g',
 				'photo' => 'images/twopeople.png',
-				'name'  => htmlentities($g['name']),
+				'name'  => htmlspecialchars($g['name']),
```
[the remark this answers is not in the document]
Comment (MrPetovan) [reply mod/acl.php]: Did you see this comment?

File: view/templates/admin/site.tpl [hunk 2]
```diff
@@ -145,7 +145,7 @@
 	{{include file="field_input.tpl" field=$dbclean_expire_conv}}
 	<div class="submit"><input type="submit" name="page_site" value="{{$submit|escape:'html'}}" /></div>
 
-	<h3>{{$worker_title}}</h3>
+	<h3>{{$worker_title|escape}}</h3>
```
Comment (MrPetovan) [inline view/templates/admin/site.tpl]: What is your reasoning to escape translations?

Comment (annando) [reply view/templates/admin/site.tpl]: It doesn't hurt.

Comment (MrPetovan) [reply view/templates/admin/site.tpl]: Fair enough, if you like more work.

Comment (annando) [reply view/templates/admin/site.tpl]: In fact it had been less work. I used "grep" to search for some patterns that contain stuff that have to be escaped. These had been ```name}}``` and _drumroll_ ```title}```. So by escaping these ones, they didn't appear when I used "grep" again to search for more.

**Comment by MrPetovan** [comment]
Part of #6208

**Pull request merged**